commands/list_clusters.go

```diff
@@ -120,7 +120,7 @@ func clustersTable(args listClustersArguments) (string, error) {
 	}
 
 	if len(response.Payload) == 0 {
-		return "", nil
+		return color.YellowString("No clusters available"), nil
```

Comment:
`available` feels like the wrong term here - makes me think that clusters do exist, but aren't accessible. Is `No clusters` enough?

Reply:
or `No clusters exist`